vpr/src/base/vpr_types.h

```diff
@@ -375,23 +375,23 @@ constexpr int NUM_PL_MOVE_TYPES = 7;
 constexpr int NUM_PL_NONTIMING_MOVE_TYPES = 3;
 
 /* Timing data structures end */
+
+// Annealing schedule
```

Comment:
Could expand this comment. AUTO_SCHED: computes initial temperature, exit criterion and temperature update rate from statistics computing during the anneal.
USER_SCHED: the user has specified explicit numbers for the key annealing schedule parameters